tests/testing.ts

```diff
@@ -68,12 +68,15 @@ export function runSolFileTest(
 ): { cairoProjects: Set<string>; result: ResultType } {
   const mangledPath = path.join(warpTest, file);
   const cairoProjects: Set<string> = new Set();
+
   try {
     transpile(compileSolFiles([file], { warnings: false }), { strict: true, dev: true }).forEach(
-      ([file, cairo]) => {
-        outputFileSync(path.join(warpTest, file), cairo);
-        createCairoProject(path.join(warpTest, file));
-        const baseDir = path.dirname(path.join(warpTest, file));
+      ([fileName, cairoCode]) => {
+        const testLocation = path.join(warpTest, fileName);
+        outputFileSync(testLocation, cairoCode);
+        createCairoProject(testLocation);
+
+        const baseDir = path.dirname(path.dirname(testLocation));
```

Comment:
Wouldn't it be better to call `path.dirname(warpTest)` here?

Reply:
I don't think so because: `warpTest = "warpTest"`. There are two dirname applications on `testLocation` because it's value is _warpTest/contract.sol/src/c.cairo_  and we want the path without the contract name and the src folder